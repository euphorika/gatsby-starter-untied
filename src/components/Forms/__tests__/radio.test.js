import React from 'React'
import renderer from 'react-test-renderer'
import Radio from '../checkbox'

describe('Radio', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Radio value="Value" name="Name" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
