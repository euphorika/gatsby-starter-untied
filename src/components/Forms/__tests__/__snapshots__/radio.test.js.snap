// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio renders correctly 1`] = `
<label
  className="label"
>
  <input
    className="element"
    name="Name"
    type="checkbox"
    value="Value"
  />
  <span
    className="afterFormElement"
  />
  <span
    className="checkmark"
  />
</label>
`;
